dashboard/app/views/admin_search/search_for_teachers.html.haml

```diff
@@ -47,7 +46,7 @@
     displayed.
     %br
 
-  #{link_to 'Download CSV', {:format => :csv}}
+  #{link_to 'Download CSV', {:format => :csv}} (commas and newlines have been replaced by spaces)
```

Comment:
technically, only the newlines in the full_address field have been replaced by spaces